### PR TITLE
Multi layers DECK.GL visualization

### LINKED ISSUE #4084: Multi-deck.gl new visualization type

### PR DESCRIPTION
**Pull request by mistercrunch**
screen shot 2017-12-19 at 2 08 20 pm

Allows users to pile up many DECK.GL slices into a single visualization
fixes https://github.com/apache/incubator-superset/issues/4084

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -1768,6 +1770,35 @@ def get_data(self, df):
...
+
+    def get_data(self, df):
+        fd = self.form_data
+        # Late imports to avoid circular import issues
+        from superset.models.core import Slice
+        from superset import db
+        slice_ids = fd.get('deck_slices')
+        slices = db.session.query(Slice).filter(Slice.id.in_(slice_ids)).all()
+        data = []
+        for slc in slices:
```
Comment (hughhhh) [inline superset/viz.py]: nit: `data = [slc for slc in slices]`

**Comment by hughhhh** [comment]
LGTM

**Comment by ahsanshah** [comment]
Wanted to confirm if auto-zoom on a multi layer Deck is not supported? Currently layering 2 Scatters does not allow for auto zoom even with conformed column name filters.

**Comment by mistercrunch** [comment]
Not currently supported as far as I know.

**Pull request merged**